pkg/api/qconfig.go

```diff
@@ -2,22 +2,28 @@ package api
 
 import (
 	"fmt"
+	"github.com/raharper/qcli"
```

Comment:
```suggestion
	"github.com/project-machine/qcli"
```

Reply:
I think the module name in the go.mod of qcli might need to be changed: https://github.com/project-machine/qcli/blob/main/go.mod will submit a PR

Reply:
Indeed!